index.html

```diff
@@ -11347,6 +11347,41 @@ <h2>A project for first-time contributions</h2>
                     </div>
                 </div>
                 <!--_____________Samarth Jain's Card END________-->
+		 
+		<!-- ________ Sonu Raghuwanshi's card START ________  -->
+                <div class="card">
+                    <p class="name">Sonu Raghuwanshi</p>
+                    <p class="contact">
+                        <i class="fab fa-twitter"></i>
+                        <a href="https://twitter.com/sonu8823812809" target="_blank"> @Sonu Raghuwanshi </a>
+                    </p>
+                    <p class="about">
+                        I am a Competitive Programming. I love problem solving. I am fond of travelling.
+                    </p>
+                    <div class="resources">
+                        <p>3 Useful Dev Resources</p>
+                        <ul>
+                            <li>
+                                <a href="#" target="_blank" title="https://www.hackerearth.com/">
```

Comment:
Please, make sure you include the URL into the href attribute, i.e.:
`<a href="https://www.hackerearth.com/" target="_blank" title="Hacker Earth">
      Hacker Earth
  </a>
`